spec.go

```diff
@@ -437,6 +437,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
```

Comment:
Yup, @LK4D4 @mrunalp and I have discussed this, and the current behavior in runc(if operator provides parent, it will append parent to Name) should be sufficient.